tests/gis_tests/test_fields.py

```diff
@@ -13,3 +14,20 @@ def test_area_field_deepcopy(self):
     def test_distance_field_deepcopy(self):
         field = DistanceField(None)
         self.assertEqual(copy.deepcopy(field), field)
+
+    def test_oracle_field_arguments(self):
+        """
+        GIS field argument preservation in migrations.
+        """
+        field = fields.PointField(
+            srid=4067,
+            dim=3,
+            extent=(50199.4814, 6582464.0358, -50000.0, 761274.6247, 7799839.8902, 50000.0),
+            tolerance=0.01
+        )
+        name, path, args, kwargs = field.deconstruct()
```

Comment:
We can ignore `name`, `path`, and `args`, e.g. `_*, kwargs = field.deconstruct()`